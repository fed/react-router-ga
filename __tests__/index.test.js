// @flow
import { create } from "react-test-renderer";
import React from "react";
import Analytics from "../";
import { MemoryRouter } from "react-router";
import { Route } from 'react-router-dom'

describe("ReactRouterGA", () => {
  beforeAll(() => {
    document.body.innerHTML = `<script src='//hello.js' />`
  })

  class Page extends React.Component {
    render () {
      return (<div>Hello</div>)
    }
  }

  it("render", () => {
    const tree = create(
      <MemoryRouter>
        <Analytics id="UA-111111111-1" debug>
          <Route component={Page} />
        </Analytics>
      </MemoryRouter>
    );
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
