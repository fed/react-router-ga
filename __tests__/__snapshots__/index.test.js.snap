// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactRouterGA render 1`] = `
<div>
  Hello
</div>
`;
